### PR TITLE
delete measure and road sign combinations

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

File: config/resources/domain.json [hunk 1]
```diff
@@ -1,188 +1,150 @@
...
-                },
-                "measureconcepts": {
-                    "predicate": "lblodmow:verkeersbordconcept",
-                    "target": "measureConcept",
-                    "cardinality": "many",
-                    "inverse": true
-                }
-            },
-            "new-resource-base": "http://data.lblod.info/verkeersbordconcepten/"
+        "sub-sign": {
```
Comment (anitacaron) [inline config/resources/domain.json:44]: sub-signs in plural here?

Comment (bdevloed) [reply config/resources/domain.json]: Thanks for spotting - fixed.

File: config/resources/domain.json [hunk 2]
```diff
@@ -1,188 +1,150 @@
...
-                }
-            },
-            "new-resource-base": "http://data.lblod.info/verkeersbordcategorieen/"
+        "related-road-sign-concepts": {
+          "predicate": "lblodmow:heeftGerelateerdVerkeersbordconcept",
+          "target": "roadSignConcept",
+          "cardinality": "many"
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/verkeersbordconcepten/"
```
Comment (Windvis) [inline config/resources/domain.json:55]: Unrelated to the other changes in this PR, but should we change this to `http://data.vlaanderen.be/id/concept/Verkeersbordconcept/`? It looks like that's how they are stored in the DB?

Comment (anitacaron) [reply config/resources/domain.json:55]: It's better to ask Niels about the different URL base they use.

Comment (bdevloed) [reply config/resources/domain.json:55]: To be confirmed with @nvdk 
Based on choice made in https://github.com/lblod/app-gelinkt-notuleren/blob/master/config/resources/master-mobiliteit.lisp

Comment (Windvis) [reply config/resources/domain.json:55]: Aha! I didn't know where we got the current ones.

**Review by Windvis — APPROVED** [review]

**Review by nbittich — APPROVED** [review]

**Pull request merged**